libkbfs/interfaces.go

```diff
@@ -1502,11 +1503,15 @@ type BareRootMetadata interface {
 	// folder.  This is only expected to be set for folder resets.
 	IsFinal() bool
 	// IsWriter returns whether or not the user+device is an authorized writer.
-	IsWriter(user keybase1.UID, deviceKID keybase1.KID) bool
+	// Note "wkb" is expected to be null for pre-v3 metadata.
+	IsWriter(user keybase1.UID, deviceKID keybase1.KID, wkb *TLFWriterKeyBundleV2) bool
```

Comment:
I like it. I was previously considering it (passing some blob) and I think it makes sense.